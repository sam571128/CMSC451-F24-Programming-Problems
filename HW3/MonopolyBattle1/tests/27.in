12 82 846
10 4
2 3
3 9
2 1
8 7
6 8
2 12
7 5
7 6
6 12
1 1
5 11
1 2
10 9
2 9
10 11
12 8
12 11
1 6
3 3
10 1
11 7
11 4
8 4
11 2
5 2
9 1
2 7
7 4
3 1
5 5
8 4
6 3
7 9
11 5
4 5
3 10
4 4
1 12
10 7
5 5
5 10
7 12
5 2
6 12
1 4
2 12
2 8
2 10
7 8
4 8
3 6
8 2
7 5
12 2
9 9
11 7
2 10
6 3
9 5
1 8
9 11
12 6
6 8
3 12
5 5
9 11
8 3
6 6
11 5
5 12
10 10
7 12
4 1
10 12
9 3
7 4
1 5
11 12
8 11
6 9
8 6
