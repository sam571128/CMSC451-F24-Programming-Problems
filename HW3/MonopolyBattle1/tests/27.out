11 13 13 37 37 19 28 39 13 37 37 39 